src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+:root {
+  --maincolor: #00acdc;
+  --header-height: 60px;
+  --header-margin: 50px;
+}
+
+* {
   margin: 0;
+  padding: 0;
+}
```

Comment:
Avoid using the universal selector (*) as it impacts performance. Set styles only for elements that require them. If you need to reset margins and paddings, consider using a more specific selector or a CSS reset file.